roles/database/tasks/main.yml

```diff
@@ -1,8 +1,12 @@
 ---
 - include: install-apt-deps.yml
-  when: initialize_local_db
+  when: fpsd_database_initialize_local_db
+
 - include: initialize-fpsd-db.yml
-  when: initialize_local_db
+  when: fpsd_database_initialize_local_db
```

Comment:
Speaking of the `initialize-test-db.yml` tasklist, re-provisioning will not have the desired result on that database if the "fpsd" database changes. This could be resolved by deleting and then re-creating the "testfpsd" template (based on the now different "fpsd" template) when changes were observed in the running of the relevant schema/table related tasks of initializing "fpsd."